test/IISIntegration.FunctionalTests/HelloWorldTest.cs

```diff
@@ -20,31 +19,23 @@ public HelloWorldTests(ITestOutputHelper output) : base(output)
         {
         }
 
-        [ConditionalTheory]
-        [OSSkipCondition(OperatingSystems.Linux | OperatingSystems.MacOSX)]
```

Comment:
Nope, just running. We build the sln (https://github.com/aspnet/KoreBuild/blob/dev/build/targets/KoreBuild.SolutionItems.targets#L7), but look for tests to run (https://github.com/aspnet/KoreBuild/blob/dev/build/targets/KoreBuild.SolutionItems.targets#L16-L18).